src/pybind/mgr/mgr_module.py

```diff
@@ -558,6 +559,28 @@ def get_daemon_status(self, svc_type, svc_id):
         """
         return self._ceph_get_daemon_status(svc_type, svc_id)
 
+    def mon_command(self, cmd_dict):
+        """
+        Helper for modules that do simple, synchronous mon command
+        execution.
+
+        See send_command for general case.
+
+        :return: status int, out std, err str
+        """
+
+        t1 = time.time()
+        result = CommandResult()
+        self.send_command(result, "mon", "", json.dumps(cmd_dict), "")
```

Comment:
I think this file lost "import json" at the top since you originally developed this, and it will need to be added back with this change.